backend/templates/consts.py

```diff
@@ -1,5 +1,7 @@
 PAYMENT_SUBJECT = 'تاییدیه پرداخت رویداد WSS'
-PAYMENT_HTML_CONTENT  = '''<!DOCTYPE html>
+RESET_PASSWORD_SUBJECT = 'WSS تنظیم مجدد رمز'
```

Comment:
Shouldn't these subjects be in English? @emranbm

Reply:
Agree. and I think it applies to other texts like email body, etc.

Reply:
done